### PR TITLE
Nasomi Fix for Lag resulting from AH seller history searches

### PR DESCRIPTION
**Pull request by Wiggo32**

### CODE REVIEW

File: sql/auction_house.sql [hunk 1]
```diff
@@ -39,6 +39,7 @@ CREATE TABLE IF NOT EXISTS `auction_house` (
   `sell_date` int(10) unsigned NOT NULL DEFAULT '0',
   PRIMARY KEY (`id`),
   KEY `itemid` (`itemid`)
```
Comment (takhlaq) [inline sql/auction_house.sql]: looks like we're missing a comma after ```(`itemid`)```

**Comment by TeoTwawki** [comment]
Build failing because it couldn't fetch packages for luacheck stuff, just fyi

**Comment by TeoTwawki** [comment]
which is blocking merge, nothing in this pr is the cause.

**Comment by TeoTwawki** [comment]
stale reviews should auto dismiss once requested changes are made :thinking:

**Pull request merged**